settings.gradle

```diff
@@ -1,7 +1,5 @@
 rootProject.name = 'terra-scientific-pipelines-service'
-include('service', 'scripts', 'client', 'rawls-client')
-
-gradle.ext.releaseVersion = "0.0.69-SNAPSHOT"
-include 'scripts:git-hooks'
```

Comment:
just curious why this was removed, not necessary?

Reply:
i just put this in the `include` line in row 2 instead